var/spack/repos/builtin/packages/rust/package.py

```diff
@@ -55,11 +56,14 @@ class Rust(Package):
         description='Install Rust source files'
     )
 
-    depends_on('cmake', type='build')
-    depends_on('python@:2.8', type='build')
+    depends_on('python@2.7:', type='build')
+    depends_on('python@2.7:2.8', when='@:1.42.0', type='build')
+    depends_on('gmake@3.81:', type='build')
+    depends_on('cmake@3.4.3:', type='build')
     depends_on('openssl')
     depends_on('libssh2')
     depends_on('libgit2')
+    depends_on('pkgconfig', type='build')
```

Comment:
Sorry, this is dumb, but could you put this next to the other build dependencies. :)